css/index.css

```diff
@@ -112,6 +144,99 @@ p {
     justify-content: center;
 }
 
+/*color the boxes*/
+.middle-content .boxes .one {
+    background-color: teal;
+}
+.middle-content .boxes .two {
+    background-color: gold;
+}
+.middle-content .boxes .three {
+    background-color: cadetblue;
+}
+.middle-content .boxes .four {
+    background-color: coral;
+}
+.middle-content .boxes .five {
+    background-color: crimson;
+}
+.middle-content .boxes .six {
+    background-color: forestgreen;
+}
+.middle-content .boxes .seven {
+    background-color: darkorchid;
+}
+.middle-content .boxes .eight {
+    background-color: hotpink;
+}
+.middle-content .boxes .nine {
+    background-color: indigo;
+}
+.middle-content .boxes .ten {
+    background-color: dodgerblue;
+}
+/*color boxes stops here */
+
+
+/*about page changes */
+.main-content {
+    margin-top: 25px;
+    display: flex;
+    flex-wrap: wrap;
+    border-top: 1px dotted black;
+    padding-top: 39px;
+    justify-content: space-between;
+}
+
+.box {
+    width: 48%;
+    margin-bottom: 25px;
+    border-bottom: 1px dotted black;
+}
+
+button {
```

Comment:
Excellent job. For aesthetics, I might suggest making the buttons a bit smaller, but functionality and animations are great.